packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1500,7 +1564,399 @@ class TextInput {
   ///   topmost [AutofillGroup] is getting disposed.
   static void finishAutofillContext({ bool shouldSave = true }) {
     assert(shouldSave != null);
-    TextInput._instance._channel.invokeMethod<void>(
+    for (final TextInputControl control in TextInput._instance._inputControls)
+      control.finishAutofillContext(shouldSave: shouldSave);
+  }
+}
+
+/// An interface for implementing text input controls.
```

Comment:
This line is a bit self-referencing, and to me it doesn't say much about what a "text input control" is. On http://api.flutter.dev/ the first paragraph is always visible and the rest are by default hidden so we want the first paragraph to be concise and informative. Maybe combine this line with the paragraph below?